validator/schemas/summary-metrics-schema-v1.json

```diff
@@ -114,7 +135,7 @@
           "eventObjectId": {
             "$id": "#/properties/query/properties/eventObjectId",
             "type": "string",
-            "enum": ["DOMAIN_IDS","ENTITY_GUIDS"]
+            "enum": ["DOMAIN_ID","ENTITY_GUIDS"]
```

Comment:
This is `DOMAIN_IDS` in the lib.

Checking around golden metrics also uses DOMAIN_IDS. And summary metrics uses DOMAIN_ID. This is probably my fault that I pluralized it in GM...

I would say to keep the plural and do the conversion in summary metrics (should already be that way)
But check the imported definitions to ensure they bring DOMAIN_IDS not DOMAIN_ID

Reply:
Right now SM will fail if we try to pass the DOMAIN_IDS plural I think, I can change that before merging this then. There are no SM in this repo that use the property btw, this is the first one.